### PR TITLE
Implement documents with multiple files

### PR DESCRIPTION
**Pull request by Glumli**

### CODE REVIEW

**Comment by Glumli** [comment]
* Add title to files
* upadte docs

File: docs/sdk.md [hunk 1]
```diff
@@ -95,9 +95,11 @@ Only the owner of the documents and those who have explicit permission from the
 
 To upload the document:
 ```javascript
- HC.uploadDocument('user_id', file, options)
+ let files = [{ data: document, title: "some_tile"}];
```
Comment (Harsimran1) [inline docs/sdk.md]: some_title*

File: docs/sdk.md [hunk 2]
```diff
@@ -162,6 +183,33 @@ In the case of any error in uploading and downloading documents, the format is:
     }
 ```
 
+#### Add files to existing docuement
+Adding files to a document requires a logged in user.
+Therefore you need to call ``addFilesToDocument`` and provide the document owners user id, the document id and the files.
+```javascript
+    let files = [{ data: document, title: "some_tile"}];
```
Comment (Harsimran1) [inline docs/sdk.md]: same

File: docs/sdk.md (hunk 1 above)
Comment (Glumli) [reply docs/sdk.md]: In my Bachelor thesis I wrote about tiles a lot... probably a reflex ;)

File: src/config/sandbox.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 const config = {
-	api: 'https://sandbox.hpihc.de',
+	api: 'https://staging.hpihc.de',
```
Comment (Harsimran1) [inline src/config/sandbox.js]: sandbox configs need to remain sandbox

**Pull request merged**